packages/elements/src/card/index.ts

```diff
@@ -132,13 +112,20 @@ export class Card extends BasicElement {
   @state()
   private footerHasContent = false;
 
+  /**
+   * Used to control aria-level for heading
+   */
+  @state()
+  private headingLevel: string | null = null;
+
   /**
    * Open menu
    * @returns {void}
    */
   private openMenu (): void {
     if (this.menuElement && !(this.menuElement.fullyOpened || this.menuElement.transitioning)) {
       this.menuElement.opened = true;
+      this.openMenuElement?.setAttribute('aria-expanded', 'true');
```

Comment:
Please do not use `setAttribute` on internal elements. Use `render` lifecycle

Reply:
Here and in all other places

Reply:
If you need to store menu opened state, you can do like this:

@state()
private menuOpened = false;

Reply:
Read my comment above....

Reply:
Fixed